01-introduction.Rmd

```diff
@@ -341,8 +341,8 @@ An important capability of **raster** and **terra** is their ability to work wit
 **raster** and **terra** also supports map algebra, as described in Section \@ref(map-algebra).
 
 In parallel with these developments of class systems and methods came the support for R as an interface to dedicated GIS software.
-**GRASS** [@bivand_using_2000] and follow-on packages **spgrass6** and **rgrass7** (for GRASS\index{GRASS} GIS 6 and 7, respectively) were prominent examples in this direction [@bivand_rgrass7_2016;@bivand_spgrass6_2016].
-Other examples of bridges between R and GIS include **RSAGA** [@R-RSAGA, first published in 2008]\index{RSAGA (package)}, **RPyGeo** [@brenning_arcgis_2012, first published in 2008], **RQGIS** [@muenchow_rqgis:_2017, first published in 2016]\index{RQGIS (package)}, and **rqgisprocess** \index{rqgisprocess (package)} (see Chapter \@ref(gis)).
+**GRASS** [@bivand_using_2000] and follow-on packages **spgrass6**, **rgrass7** and **rgrass** (for GRASS\index{GRASS} GIS 6, 7, 7 + 8 respectively) were prominent examples in this direction [@bivand_rgrass7_2016;@bivand_spgrass6_2016].
```

Comment:
I suggest only mentioning one of the **rgrass** interfaces to avoid confusion. Which is most likely to be most appropriate for most people?

Reply:
Definitely **rgrass** then; it is for 'GRASS 7+'. See Readme of https://github.com/rsbivand/rgrass.

Reply:
Let's remove the others. Already too many packages mentioned in this book, hard to know which to use just with recommended ones!

Reply:
@Robinlovelace on the other hand this section describes _history_ of R-spatial, so that may still be a reason to mention **spgrass6**, **rgrass7** as well as **RQGIS** in here. Adding more explicitly that these are superseded by **rgrass** & **qgisprocess** may be more appropriate?

Reply:
@florisvdh yes -- that would be the good approach here